lib/components/parallax_component.dart

```diff
@@ -78,21 +78,20 @@ class ParallaxLayer {
 
     // Number of images that can fit on the canvas plus one
     // to have something to scroll to without leaving canvas empty
-    final count = Vector2.all(1) + size.clone()
-      ..divide(_imageSize);
+    final Vector2 count = Vector2.all(1) + (size.clone()..divide(_imageSize));
```

Comment:
The parenthesis here is what was missing and which completely broke the positioning.

Reply:
I think the parenthesis would only be necessary in the multiline case right? But either way lets keep it for clarity. A bit odd we are summing (1,1) to the vector? looks a bit unusual to me.

Reply:
It is explained in the comment above :)
Does dart make a difference of whitespace for ordering?!